src/components/Loader/PeopleTable.tsx

```diff
@@ -0,0 +1,127 @@
+import classNames from 'classnames';
+import React from 'react';
+
+import { Link } from 'react-router-dom';
+
+import { Loader } from './Loader';
+
+import { Person } from '../../types';
+
+type Props = {
+  peoples: Person[],
+  nickName: string,
+  loading: boolean,
+  error: boolean,
+};
+
+export const PeopleTable: React.FC<Props> = ({
+  peoples,
+  nickName,
+  loading,
+  error,
+}) => {
+  const isSelected = (people: Person) => nickName === people.slug;
+  const isWoman = (sex: string) => sex === 'f';
+
+  const handleParentName = (parentName: string | null) => {
+    const parent = peoples.find(people => people.name === parentName);
+
+    if (parent) {
+      return (
+        <Link
```

Comment:
It would be nice to have separate component for this element, it's repeated several times